repository-data/webfiles/src/main/resources/site/scss/components/_navigation-tiles.scss

```diff
@@ -0,0 +1,138 @@
+.navigation-section {}
+
+.navigation-section__introduction {
+    margin-bottom: 20px;
+}
+
+.navigation-section--column .navigation-section__tiles {
+    @include media(desktop) {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-between;
+    }
+}
+
+.navigation-tile,
+.navigation-tile:visited,
+.navigation-tile:link {
+    display: block;
+    margin-bottom: 20px;
+    padding: 40px 40px 60px 40px;
+    border: 3px solid transparent;
+    color: $text-colour;
+    text-align: center;
+    text-decoration: none;
+    border-radius: 12px;
+    box-shadow: 0 0 15px 0 rgba($black,0.15);
+    background: linear-gradient(56deg, $grey-5 9%, $white 100%) center;
+    // Make bg bigger to counteract transparent border pushing it in
+    background-size: calc(100% + 6px) calc(100% + 6px);
+}
+
+.navigation-tile--column {
+    @include media(desktop) {
+        flex-grow: 0;
+        flex-shrink: 0;
+        flex-basis: auto;
+        width: calc(50% - (25px / 2));
+    }
+}
+
+.navigation-tile--full {
+    @include media(tablet) {
+        display: flex;
+    }
+}
+
+.navigation-tile:hover,
+.navigation-tile:active,
+.navigation-tile:focus {
+    background: linear-gradient(56deg, $grey-5 9%, $white 100%);
+    box-shadow: 0 0 15px 0 rgba($black,0.15);
+    outline: none;
+}
+
+.navigation-tile:hover {
+    border-color: $yellow-link-hover;
+}
+.navigation-tile:focus,
+.navigation-tile:active {
+    border-color: $yellow-link-active;
+}
+
+.navigation-tile__image {
+    margin-right: auto;
+    margin-bottom: 20px;
+    margin-left: auto;
+}
+.navigation-tile--full .navigation-tile__image {
+    @include media(tablet) {
+        height: 76px;
+        margin-right: 40px;
+        margin-left: 0;
+    }
+}
+
+.navigation-tile__image--icon {
+    max-height: 76px;
+}
+
+.navigation-tile__content {}
+
+.navigation-tile--full .navigation-tile__content {
+    @include media(tablet) {
+        max-width: 528px;
+        text-align: left;
+    }
+}
+
+.navigation-tile__title {
+    font-size: 22px;
+    line-height: 1.32;
+    font-weight: bold;
+    color: currentColor;
```

Comment:
I am sure it's fine, but please can you talk me through this? I had thought SCSS variables begin with a '$' dollar sign.

Reply:
Today's your lucky day! This is a great CSS feature.

`currentColor` has had full browser support since IE9: https://caniuse.com/#feat=currentcolor. It's a way of using the currently defined value of `color` for other properties.

See https://codepen.io/jodi/pen/VwvQRYW for a silly example

Reply:
That is a great feature 👍 Thanks for letting me know.